src/components/structures/RoomView.js

```diff
@@ -77,13 +77,6 @@ module.exports = React.createClass({
     },
 
     componentWillUnmount: function() {
-        if (this.refs.messagePanel) {
-            var messagePanel = ReactDOM.findDOMNode(this.refs.messagePanel);
-            messagePanel.removeEventListener('drop', this.onDrop);
-            messagePanel.removeEventListener('dragover', this.onDragOver);
-            messagePanel.removeEventListener('dragleave', this.onDragLeaveOrEnd);
-            messagePanel.removeEventListener('dragend', this.onDragLeaveOrEnd);
-        }
```

Comment:
The FB example is spurious - that's a listener on 'window', which *isn't* (necessarily) being freed.

The source of concern was that if we needed to deal with the messagePanel appearing during the lifetime of the component, maybe we also needed to deal with it *disappearing* - so it is somewhat comforting if there is no magic teardown code for the messagePanel.

Your point is valid though. I'll add it back in with a comment.